test/common.js

```diff
@@ -124,61 +124,98 @@ describe('common', () => {
 
 	describe('document mutations', () => {
 		it('should update language if "lang" gets set', (done) => {
-			documentLocaleSettings.addChangeListener(() => {
+			const cb = () => {
```

Comment:
Since `afterEach` was resetting the attributes, these listeners were getting called again, causing an error about `done` being called multiple times.